lib/overcommit/hook/post_checkout/git_lfs.rb

```diff
@@ -0,0 +1,20 @@
+module Overcommit::Hook::PostCheckout
+  # Invokes Git LFS command that checks status of lockable files tracked by Git LFS
+  #
+  # @see https://git-lfs.github.com/
+  class GitLfs < Base
+    def run
+      unless in_path?('git-lfs')
+        return :warn, 'This repository is configured for Git LFS but \'git-lfs\' ' \
+        "was not found on your path.\nIf you no longer wish to use Git LFS, " \
+        'disable this hook by removing or setting \'enabled: false\' for GitLFS ' \
+        'hook in your .overcommit.yml file'
+      end
```

Comment:
You shouldn't need to perform this check explicitly. Simply specify the `required_executable` option in the hook configuration.

I can see that you're trying to provide a bit extra information, but my hope is that the existing error message is pretty straightforward, and that anyone working with a repo using LFS knows what Git LFS is.